docs/src/main/sphinx/installation/cli.rst

```diff
@@ -29,6 +29,22 @@ The CLI uses the HTTP protocol and the
 :doc:`Trino client REST API </develop/client-protocol>` to communicate
 with Trino.
 
+TLS/HTTPS
+---------
+
+Typically a Trino server is available with a HTTPS URL, so all network traffic
+from the CLI uses TLS. This :doc:`TLS configuration </security/tls>` is common,
+since it is a requirement for any authentication.
+
+Connecting to the server simply requires you to use the HTTPS URL:
+
+.. code-block:: text
+
+    ./trino --server https://trino.example.com
+
+No other options are necessary for the recommended approach of using a globally
+trusted certificate.
+
```

Comment:
Try "A globally trusted certificate is the standard approach, no other options are necessary."

Reply:
That implies a different meaning..  reworded a bit differently to clarify.